app/src/main/java/com/battlelancer/seriesguide/provider/SgShow2Helper.kt

```diff
@@ -94,6 +94,12 @@ interface SgShow2Helper {
     @Query("SELECT _id, series_status, series_next, series_runtime FROM sg_show")
     fun getStats(): List<SgShow2Stats>
 
+    @Query("SELECT count(series_id) FROM (SELECT series_id, series_status, sum(case when episode_watched = '0' then 1 else 0 end) as episodes_unwatched FROM sg_episode LEFT OUTER JOIN sg_show ON sg_episode.series_id = sg_show._id GROUP BY series_id) WHERE episodes_unwatched = 0 AND series_status IN (0, 3)")
```

Comment:
As I'm a SQL noob: is the GROUP BY a performance optimization? Or am I missing something?